PROGRESS/alex.md

```
@@ -6,6 +6,39 @@
 
 ---
 
+## 2026-08-29 · docs/alex-poll-answers · полный проход опросником по открытым вопросам
+
+**Сделано:** опросник 2026-08-29 закрыл 25 решений восемью блоками:
+архитектура (§8 №1–4), инфраструктура (#39 Resend, #40 ключ в env,
+#46 выровнять имя базы), деньги (#22, #27, #28, #47), склад и финансы
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)